code/modules/mob/living/silicon/robot/laws.dm

```diff
@@ -28,6 +29,7 @@
 
 	who << "<b>Obey these laws:</b>"
 	laws.show_laws(who)
+	// TODO: Update to new antagonist system.

```

Comment:
Left them in as I intend to port over Bay's antagonist system in the very near future (as it allows for any round to be mixed with any other round: e.g. vampire + nuke ops).